Marlin/Configuration_adv.h

```diff
@@ -1583,6 +1583,13 @@
     //#define DOUBLE_LCD_FRAMERATE        // Not recommended for slow boards.
   #endif
 
+  //#define BEEP_ON_SCROLL                // Beep when any menu is scrolled
+  #if ENABLED(BEEP_ON_SCROLL)
+    #define SCROLL_BEEP_DURATION   2
+    #define SCROLL_BEEP_FREQUENCY  15000
+  #endif
```

Comment:
What kind of UI or accessibility standard is this? Wouldn't it make more sense to make a subtle "click" sound every time a new menu item is selected or every time a value changes in response to the encoder, rather than just when the screen scrolls? And then you would want some distinctly different feedback when the user reaches the end of the menu or the limit of the value range. Note that for devices without a proper `SPEAKER` the sound produced will not be a subtle click but a loud beep.

Reply:
Right, maybe "scroll" is the wrong word. The effect is a subtle "click" when the selected menu line changes, exactly as you describe.

What name do you suggest? Maybe BEEP_ON_MENU_LINE_CHANGE?

Regarding the loud beep, that's why I made the duration so short, but you are right that is specific to the buzzer/speker type.

Reply:
The implementation only applied to scrolling. Save this feature for the future. This patch to encoder behavior is good in its current state. I find no issues in my testing with a GLCD.

Reply:
Great!
With *save for later* you mean I should open a separate PR for the scroll audio feedback?